spec/requests/api/merge_requests_spec.rb

```diff
@@ -219,10 +222,10 @@
 
       it "should return merge_request" do
         post api("/projects/#{fork_project.id}/merge_requests", user2),
-        title: 'Test merge_request', source_branch: "stable", target_branch: "master", author: user2, target_project_id: project.id, description: 'Test description for Test merge_request'
+        title: "Test merge_request", source_branch: "stable", target_branch: "master", author: user2, target_project_id: project.id, description: "Test description for Test merge_request"
```

Comment:
Line is too long. [187/80]<br>Align the parameters of a method call if they span more than one line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.